sorting/radix_sort/radix_sort.cpp

```diff
@@ -0,0 +1,96 @@
+/*
+    Radix Sort Algorithm
+    ----------------------
+
+    Time Complexity - O(d*(n + b)) 
+        where d is the max number of digits, n is the size of the input & b is the base of the system
+
+    Space Complexity - O(n + b)
+        where n is the size of the input & b is the base of the system
+*/
+
+#include <iostream>
+#include <cstdlib>
+#include <ctime>
+
+using namespace std;
+
+// Gets the maximum element in array (To find the max number of digits)
+int getMax(int arr[], int num) {
```

Comment:
what if array's empty. May happen!

Reply:
Should I just check if the array is empty and then exit the program if it is or should I use exception handling?